src/js/types/generic.type.ts

```diff
@@ -0,0 +1 @@
+export type GenericObjectType = { [key: string]: any };
```

Comment:
makes sense to have something like this in a types folder :+1: setting a note here for other defined types as they should be present in the file where the logic is declared

Reply:
Code looks looks great! Giving the PR an approve, but this line gives me some pause. Is this for the no implicit any rule? Def agree with your comment below @mamaddox, and if we're already creating workarounds I think it is best to remove the rule altogether and replace it with no explicit any. If the issue is to avoid typing values and returns as `any`, no explicit any will accomplish this without the weird side-effects.

Reply:
Yeah. This is for that `noImplicitAny` rule.

I just want to bump this comment: https://github.com/jwplayer/jwplayer/pull/3541#discussion_r359961348

> To note, there is the '[suppressImplicitAnyIndexErrors'](https://www.typescriptlang.org/docs/handbook/compiler-options.html) tsconfig compiler option

This setting would suppress these `noImplicitAny` errors when indexing objects (which we have already pointed out twice). Something to keep in mind.

Reply:
Keeping for now 👍